examples/email-custom-handler.php

```diff
@@ -15,7 +15,7 @@ class SimpleEmailHandler extends EmailHandler
 {    
     public function handle(NotificationInterface $notification)
     {
-        mail($notification->getRecipientAddress(), $notification->subject, $notification->body);
+        mail($notification->getRecipientAddresses(), $notification->subject, $notification->body);
```

Comment:
Yes, I just checked it now, you pass a string with comma separated email addresses.
I'm going to modify it.